src/combobox/combobox.component.ts

```diff
@@ -402,6 +417,11 @@ export class ComboBox implements OnChanges, AfterViewInit, AfterContentInit {
 	 * Sets the list group filter, and manages single select item selection.
 	 */
 	public onSearch(searchString) {
+		if (searchString && this.type === "single") {
+			this.showCloseButton = true;
+		} else {
+			this.showCloseButton = false;
```

Comment:
Gotta update all the references 🙂 FWIW `F2` on a symbol in vscode generally lets you automatically rename all the references

Reply:
(but not the ones in the template ... yet)

Reply:
🤦‍♂ thats my bad

Reply:
No worries!